packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -10,8 +10,8 @@
                 :class="$style['c-address-formField']"
                 name="address-line-1"
                 maxlength="100"
-                :label-text="$t('labels.line1')"
-                :placeholder="$t('labels.line1')"
+                :label-text="$t('formFields.address.line1.label')"
```

Comment:
Could we use some variables so that we don't need to type out the full qualifications each time?

Reply:
This will be done in the next pr https://github.com/justeat/fozzie-components/compare/f-checkout-translations...f-checkout-guest-form-field?expand=1